WordPress/Classes/ViewRelated/Aztec/Renderers/SpecialTagAttachmentRenderer.swift

```diff
@@ -70,12 +73,15 @@ extension MoreAttachmentRenderer: TextViewAttachmentImageProvider {
 
 // MARK: - Constants
 //
-extension MoreAttachmentRenderer {
+extension SpecialTagAttachmentRenderer {
 
     struct Constants {
         static let defaultDashCount = CGFloat(8.0)
         static let defaultDashWidth = CGFloat(2.0)
         static let defaultHeight = CGFloat(44.0)
-        static let defaultCommentText = "more"
+    }
+
+    struct Tags {
+        static let supported = ["more", "nextpage"]
```

Comment:
Why not to use a String Enum?

Reply:
Thought of that! but we'd need to have an array of `supported` tags anyways:

```
enum SupportedTags: String {
   case more = "more"
   case nextpage = "nextpage"

   static let all = [.more, .nextpage]
}
```

Which for this use case felt like an overkill, since we only use this in a single spot.